Gemfile

```diff
@@ -11,3 +11,4 @@ end
 gem 'jruby-openssl', :platform=> :jruby
 gem 'activemodel', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
 gem 'linkeddata', '~> 1.99'
+gem 'rake', '< 12'
```

Comment:
Well, I wanted to use AF 8, but discovered that the bump to ActiveTriples 0.4.x was a problem, so was trying to go this route.